pkg/controllers/awsloadbalancercontroller/status.go

```diff
@@ -16,16 +16,21 @@ import (
 )
 
 const (
-	DeploymentAvailableCondition         = "DeploymentAvailable"
-	DeploymentUpgradingCondition         = "DeploymentUpgrading"
-	CredentialsRequestUpdatingCondition  = "CredentialsRequestUpgrading"
-	CredentialsRequestAvailableCondition = "CredentialsRequestAvailable"
+	DeploymentAvailableCondition = "DeploymentAvailable"
+	DeploymentUpgradingCondition = "DeploymentUpgrading"
+	CredentialsRequestAvailable  = "CredentialsRequestAvailable"
```

Comment:
This constant should still be called `CredentialsRequestAvailableCondition`.

Reply:
Fixed